lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/EvidenceFactory.java

```diff
@@ -84,99 +78,67 @@ public Object[] create(KBVItem kbvItem, EvidenceRequest evidenceRequest)
         return new Map[] {objectMapper.convertValue(evidence, Map.class)};
     }
 
-    private CheckDetail[] createCheckDetailsWithKbvQuality(KBVItem kbvItem)
-            throws JsonProcessingException {
-        var questionState = objectMapper.readValue(kbvItem.getQuestionState(), QuestionState.class);
+    private CheckDetail[] withKbvQuality(KBVItem kbvItem) throws JsonProcessingException {
+        QuestionState questionState =
+                objectMapper.readValue(kbvItem.getQuestionState(), QuestionState.class);
+        KbvQuestionAnswerSummary answerSummary = kbvItem.getQuestionAnswerResultSummary();
 
-        if (hasPassedWithOneIncorrectAnswer(kbvItem)) {
+        if (hasPassedWithOneIncorrectAnswerAfterTwoBatches(kbvItem, questionState)) {
```

Comment:
I'm keen to make the function names as short as possible what do you think about something like this to make use of numbers/symbols that help to convey meaning? (hoping that makes sense!)
```
if (batchNumberOfIncorrectAnswer(kbvItem, questionState) == 2)
```